internal/support/cronscheduler/infrastructure/manager.go

```diff
@@ -210,15 +210,13 @@ func (m *manager) addNewJob(job models.ScheduleJob) errors.EdgeX {
 						} else {
 							jobScheduledAt = nextRun.UnixMilli()
```

Comment:
use the previousTime in `AfterJobRuns`
don't set any variable in `BeforeJobRuns` anymore, and just log the debug logs here